packages/javascript/bh-shared-ui/src/components/HelpTexts/CodeController/CodeController.tsx

```diff
@@ -0,0 +1,137 @@
+import { Button, Typography } from '@mui/material';
```

Comment:
This is a new component for helping with displaying and copying large code snippets.